ksql-common/src/main/java/io/confluent/ksql/function/types/ParamTypes.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.types;
+
+public final class ParamTypes {
+
+  private ParamTypes() {
+  }
+
+  public static final BooleanType BOOLEAN = new BooleanType();
```

Comment:
I think it would be nicer to have these on the specific classes.

Reply:
For this one I was following the pattern in `SqlTypes` - I'll put it in both and have this one reference the one in the specific class